core/src/main/java/nl/nn/adapterframework/util/Misc.java

```diff
@@ -1413,4 +1413,8 @@ public static String jsonPretty(String json) {
 
 		return sw.toString().trim();
 	}
+
+	public static String toUpperCamelCase(String arg) {
+		return arg.substring(0,  1).toUpperCase() + arg.substring(1);
+	}
```

Comment:
Dit is initialCaps, en kan nu niet overweg met upper case strings

Reply:
Ik wilde toUpperCamelCase niet veranderen. Die gebruik ik veel in de Frank!Doc. Is het zo OK?